include/deal.II/lac/la_parallel_block_vector.templates.h

```diff
@@ -421,6 +421,16 @@ namespace LinearAlgebra
 
 
 
+    template <typename Number>
+    void
+    BlockVector<Number>::set_ghost_state(const bool ghosted) const
+    {
+      for (unsigned int block = 0; block < this->n_blocks(); ++block)
+        this->block(block).set_ghost_state(ghosted);
```

Comment:
Though I now see that this is apparently something the base vectors already do...